raiden-ts/src/messages/utils.ts

```diff
@@ -313,25 +313,25 @@ export function getBalanceProofFromEnvelopeMessage(
 
 /**
  * Encode a Message as a JSON string
- * Uses lossless-json to encode BigNumbers as JSON 'string' type, as Raiden
+ * Uses io-ts codec to encode BigNumbers as JSON 'string' type, as Raiden
```

Comment:
Yes, it does, because this is how they know what the output will be: in this case, they can go for the BigNumberC, and see it encodes BNs as decimal strings, so this isn't just _serialize however you can_, but instead is an important part of the protocol.